### DISCUSSION TITLE
Is this possible-- note repeating (for drum ratcheting)?

### DISCUSSION THREAD

**Discussion by dindrick** [shanteacontrols/OpenDeck#242] [General]
So I’ve had this idea/desire for a midi controller for awhile—
I want a small device that basically adds in-time drum ratcheting for drum machines (octatrack, digitakt, etc. ). For my current live setup, I use a Drumbrute impact which has this, but I am going to be removing it from my stop to free up inputs on my octatrack.

It would work similarly to how note repeat works on an mpc, though with the caveat that I want to design it for one handed operation, since I often am playing a keyboard or using the OT slider with my right hand. NOTE- I have been able to get this working with a beatstep pro, but that takes up a lot more real estate than I would like.

Some requirements (just sort of thinking out loud here):

- latching (on or off) of repeat rate (so you don’t have to hold down both the rate selector as well as the note selector)
- Somehow able to choose to repeat notes at 1/4, 1/8, 1/16, 1/32 
- midi in and out, and obviously able to receive midi start, tempo and send midi notes at specific intervals in relation to tempo in

Is this possible on OpenDeck, or would there be interest in adding this?

Thank you!

**Comment by paradajz** [comment]
It's not possible currently and also not planned. It's a lot of work really.
